src/components/SearchPage/SearchPage.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import HeaderContainer from "../HeaderContainer/HeaderContainer";
+import SearchResultsContainer from '../SearchResultsContainer/SearchResultsContainer';
+
+const SearchOptions = {
+    first: 'Title',
+    second: 'Genre'
+};
+
+export default class SearchPage extends React.PureComponent {
+    constructor(props) {
+        super(props);
+        this.state = {
+            searchOption: SearchOptions.first,
+            searchValue: '',
+            searchResults: this.props.films,
+        };
+        this.updateSearchOption = this.updateSearchOption.bind(this);
+        this.getMovies = this.getMovies.bind(this);
+    }
+
+    updateSearchOption() {
+        const { searchOption, } = this.state;
+        const newValue = (searchOption === SearchOptions.first) ? SearchOptions.second : SearchOptions.first;
```

Comment:
I suggest do not hardcode rule here and make it more flexible

Reply:
Didn't do it at this iteration